fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -1842,6 +1844,23 @@ public void setMaxExecutionTimeMS(String maxExecutionTimeMS) {
         this.queryTimeoutS = this.maxExecutionTimeMS / 1000;
     }
 
+    public void checkPipelineTaskNum(String value) throws Exception {
+        checkFieldValue("parallel_pipeline_task_num", 0, value);
```

Comment:
It is a setter method.
You should do :
1. check if the value is valid
2. parse and set the value